lib/src/generator/generator_utils.dart

```diff
@@ -34,6 +34,16 @@ String generateCategoryJson(Iterable<Categorization> categories, bool pretty) {
   return encoder.convert(indexItems.toList());
 }
 
+String removeHTMLTags(String? input) {
+  if (input == null) {
+    return '';
+  }
+  RegExp exp = RegExp(r"<[^>]*>", multiLine: true, caseSensitive: true);
```

Comment:
We can give this regular expression a name that denotes what it matches, something like "tag" or "htmlTag".

Reply:
Addressed in new commit